modules/big-richmediaBidAdapter.js

```diff
@@ -112,11 +112,6 @@ export const spec = {
     return baseAdapter.getUserSyncs(syncOptions, responses, gdprConsent);
   },
 
-  transformBidParams: function (params, isOpenRtb) {
-    if (!baseAdapter.transformBidParams) { return params; }
-    return baseAdapter.transformBidParams(params, isOpenRtb);
-  },
-
```

Comment:
Just to confirm, this adapter extends [appnexusBidAdapter](https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js) which uses this function. Was this taken into consideration?